homework/index.js

```diff
@@ -30,17 +35,122 @@
     return elem;
   }
 
+  function addRow(tBody, label, value) {
+    const row = createAndAppend('tr', tBody);
+    if (value === null) {
+      createAndAppend('td', row, { text: `${label} :`, class: 'label' });
+      createAndAppend('td', row, {
+        text: `No ${label.toLowerCase()} is provided for this repository.`,
+      });
+    } else {
+      createAndAppend('td', row, { text: `${label} :`, class: 'label' });
+      createAndAppend('td', row, { text: value });
+    }
+    return row;
+  }
+
+  function jsUcfirst(string) {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+  }
+
+  function createDetails(leftColumn, repository) {
+    const table = createAndAppend('table', leftColumn, { id: 'main-table' });
+    const tBody = createAndAppend('tbody', table);
+    const firstRow = addRow(tBody, 'Repository', '');
+    createAndAppend('a', firstRow.lastChild, {
+      href: repository.html_url,
+      target: '_blank',
+      text: jsUcfirst(repository.name),
+    });
+    addRow(tBody, 'Description', repository.description);
+    addRow(tBody, 'Forks', repository.forks);
+    addRow(tBody, 'Updated', repository.updated_at);
+  }
+
+  function createContributors(rightColumn, requestURL) {
+    fetchJSON(requestURL)
+      .then(contributors => {
+        const rightUL = createAndAppend('ul', rightColumn, { id: 'main-list' });
+        contributors.forEach(contributor => {
+          const contributorItem = createAndAppend('li', rightUL, {
+            class: 'contributor-block',
+          });
+          const contributorWrapper = createAndAppend('div', contributorItem, {
+            class: 'contributor-wrapper',
+          });
+          createAndAppend('img', contributorWrapper, {
+            src: contributor.avatar_url,
+            class: 'contributor-avatar',
+            alt: `Avatar of ${contributor.login}`,
+          });
+
+          createAndAppend('a', contributorWrapper, {
```

Comment:
It would be nice if clicking anywhere on the list item would take you to the contributor's page. You can achieve this by wrapping the entire inner content of the `li` into an anchor tag with an appropriate `href` and replacing your current anchor tag with just a `div` or `p` or something like that.